app/models/word.rb

```diff
@@ -0,0 +1,6 @@
+class Word < ApplicationRecord
+  validates :category_id, presence: true
+  validates :word, uniqueness: true, presence: true
```

Comment:
> **validates :word, uniqueness: true, presence: true**

We changed the name of the column to WORDS NAME instead of WORDS WORD
![image](https://user-images.githubusercontent.com/35232348/104156197-92e4f880-5423-11eb-91c9-98dbf57b16eb.png)
We have an error like this because it was looking for WORDS WORD, so change this to **validates :name**